docs/source/conf.py

```diff
@@ -287,6 +283,30 @@ def process_modin_accessors(args):
             return list(map(process_modin_accessors, items))
         return items
 
+def process_signature(app, what, name, obj, options, signature, return_annotation):
+
+    # Names to remove from signature (AST related):
+    names_to_remove = ['_emit_ast', '_ast']
+
+    def remove_from_signature(signature, name_to_remove):
+        if name_to_remove not in signature:
+            return signature
+
+        if signature.startswith('(') and signature.endswith(')'):
+            # temporarily remove parentheses, add after removing name_to_remove parts.
+            signature = signature[1:-1]
+            parts = [p for p in signature.split(',') if name_to_remove not in p]
+            signature = ','.join(parts)
+
+            return '(' + signature + ')'
+        else:
+            return signature
+
+    if signature and any(name_to_remove in signature for name_to_remove in ['_emit_ast', '_ast']):
```

Comment:
are we doing the same check twice to see if `_emit_ast`, `_ast` is in signature. I don't think this is necessary. We can remove the check here and make things simpler and easier to read.